src/System.Devices.Gpio/publish.cmd

```diff
@@ -0,0 +1,23 @@
+@echo off
```

Comment:
Why is the publish command within a batch script (.cmd) but the unpublish in a bash script (.sh)?

Reply:
This is because the publish command is intended to run on the developers machine while the unpublish script has to run on the device itself.

Reply:
The publish can come from a Linux box too, so a .sh is also welcome.

Reply:
Those scripts and the Samples project are just for me, so I can easily try and test the API. They are not part of the API and are not intended to be used by users.